jquery.fullPage.js

```diff
@@ -2076,22 +2080,33 @@
         function resizeHandler(){
             //checking if it needs to get responsive
             responsive();
+            if(document.body.classList.contains('fp-responsive')) return; // IVO GELOV
 
             // rebuild immediately on touch devices
             if (isTouchDevice) {
                 var activeElement = $(document.activeElement);
 
                 //if the keyboard is NOT visible
                 if (!activeElement.is('textarea') && !activeElement.is('input') && !activeElement.is('select')) {
-                    var currentHeight = $window.height();
+                    var currentHeight = viewportSize.getHeight(); // IVO GELOV = $window.height();
 
                     //making sure the change in the viewport size is enough to force a rebuild. (20 % of the window to avoid problems when hidding scroll bars)
-                    if( Math.abs(currentHeight - previousHeight) > (20 * Math.max(previousHeight, currentHeight) / 100) ){
-                        reBuild(true);
-                        previousHeight = currentHeight;
+                    if( Math.abs(currentHeight - previousHeight) > (20 * Math.max(previousHeight, currentHeight) / 100) ) reBuild(true); // IVO GELOV
+                    else
+                    {

```

Comment:
If the change is less than 20%, we should not ignore the event - just postpone the rebuild for some time,